.github/workflows/build.yml

```diff
@@ -96,7 +96,7 @@ jobs:
       - # === Install Deps ===
         name: Install Deps
         shell: bash
-        timeout-minutes: 1
+        timeout-minutes: 10
```

Comment:
Do you have an instance where install deps took more than 1 minute? I'm asking because I never found one myself and I'd rather be notified after 1 minute to retry a failing job than after 10 minutes when I think it's happily running integration tests.

Reply:
Yes, my previous PR was consistently failing on Windows because a `go install` step was taking longer than 1 minute.